### PR TITLE
Add Debian to installation.md

### PR DESCRIPTION
**Pull request by brrock**
Simply add Debian as there is a package in the SID repos which I am daily driving is working great

### CODE REVIEW

File: pages/Getting Started/Installation.md [hunk 1]
```diff
@@ -92,6 +92,21 @@ If you are on an older version of Fedora, you can also compile it yourself by
 following the instructions
 [here](https://github.com/hyprwm/Hyprland/discussions/284)
 
+{{% /details %}}
+{{% details title="Debain*" closed="true" %}}
```
[the remark this answers is not in the document]
Comment (brrock) [reply pages/Getting Started/Installation.md]: Should be resolved

File: pages/Getting Started/Installation.md [hunk 2]
```diff
@@ -92,6 +92,21 @@ If you are on an older version of Fedora, you can also compile it yourself by
...
+
+```sh
+sudo apt  install hyprland
+```
+Note: Soon will be in the trixie repos, but as Hyprland is bleeding edge it is only recommend to install with SID even if any other repo got hyprland.
+
+Alternatively, you can also follow the instructions under
+["Manual (Manual Build)"](#manual-manual-build) to build Hyprland yourself.
+
+Note: _Hyprland is not available for Bookworm as it packages are too old
```
Comment (fufexan) [inline pages/Getting Started/Installation.md:108]:
```suggestion
{{< callout type=info >}}

Hyprland is not available for Bookworm as its packages are too old.

{{< /callout >}}
```

**Review by fufexan — CHANGES_REQUESTED** [review]

**Review by brrock — COMMENTED** [review]
Sorry for the typos I was on mobile

**Comment by brrock** [comment]
@vaxerski

**Review by fufexan — APPROVED** [review]

**Pull request merged**